pkg/ccl/backupccl/create_scheduled_backup.go

```diff
@@ -785,11 +806,29 @@ func createBackupScheduleHook(
 	if !ok {
 		return nil, nil, nil, false, nil
 	}
+
 	eval, err := makeScheduledBackupEval(ctx, p, schedule)
 	if err != nil {
 		return nil, nil, nil, false, err
 	}
 
+	scheduleLabel, err := eval.scheduleLabel()
+	if err != nil {
+		return nil, nil, nil, false, err
+	}
+
+	exists, err := checkScheduleAlreadyExists(ctx, p, scheduleLabel)
+	if err != nil {
+		return nil, nil, nil, false, err
+	}
+
+	if exists {
+		p.BufferClientNotice(ctx,
+			pgnotice.Newf("schedule %q already exists, skipping", scheduleLabel),
+		)
+		return nil, nil, nil, false, nil
```

Comment:
Shouldn't this be an error? You specified `IF NOT EXIST` and it exists -- seems like we should error.

If you agree, you could probalby just return an error from check schedule already exists.

Reply:
@miretskiy I spoke with @adityamaru about this and we think we should keep this as is. First, I believe `CREATE TABLE IF NOT EXISTS` follows the same pattern. Also, I think the conceptual reasoning is the following: if the schedule **does** exist and the user specifies `IF NOT EXIST`, the user would expect the `CREATE SCHEDULE` query to be skipped, rather than returning an error that the user has to handle. (@adityamaru  correct me if i'm wrong). Does that make sense?

Reply:
That makes perfect sense.

Reply:
great. resolved.